Objects/object.c

```diff
@@ -13,6 +13,7 @@
 #include "pycore_memoryobject.h"  // _PyManagedBuffer_Type
 #include "pycore_namespace.h"     // _PyNamespace_Type
 #include "pycore_object.h"        // PyAPI_DATA() _Py_SwappedOp definition
+#include "pycore_optimizer.h"     // _PyCounterExecutor_Type, _PyCounterOptimizer_Type, _PyDefaultOptimizer_Type, _PyUOpExecutor_Type, _PyUOpOptimizer_Type
```

Comment:
This line's a tad long, maybe just something like:

```suggestion
#include "pycore_optimizer.h"     // _PyUOpExecutor_Type, _PyUOpOptimizer_Type, ...
```

TBH I've never been a huge fan of this convention (it gets stale pretty quickly), but it's prevalent in the codebase so...